roles/network-multus/templates/ovs-vsctl.yml

```diff
@@ -0,0 +1,53 @@
+---
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ origin_node_provisioner_name }}
+  namespace: {{ namespace }}
+  labels:
+    tier: node
+    app: ovs-cni-plugin
+spec:
+  template:
+    metadata:
+      labels:
+        tier: node
+        app: ovs-cni-plugin
+    spec:
+      nodeSelector:
+        beta.kubernetes.io/arch: amd64
+      tolerations:
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      containers:
+      - name: ovs-vsctl
+        command:
+        - /bin/bash
+        - -c
+        - |
+          if [ ! -f /host/usr/bin/ovs-vsctl ]; then
```

Comment:
if you check that ovs is not installed on openshift nodes, why do we need this check?

Reply:
where do I check that now?

It was at one point in this PR, but then we realized the relevant ansible code (in provision.yml) only runs on master, so checking if ovs is installed was redundant, and was removed.

Reply:
I see. Ok, no problem then